### ISSUE TITLE
schema generator should default the 'kind' property to the name of the class

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/fabric8#3593]
e.g.
```
Pod pod = new Pod();
assertThat(pod.getKind()).equals("Pod");
```

**Comment by davsclaus** [comment]
If I edit the model file, I can get the generator to add a default value
https://github.com/fabric8io/fabric8/blob/master/components/kubernetes-api/src/main/kubernetes/api/doc/kube-schema.json#L902

And add a default value
```
            "kind":{  
               "type":"string",
               "default":"Pod"
            },
```

Then the generated code has that 
```
    @JsonProperty("kind")
    private java.lang.String kind = "Pod";
```

**Comment by davsclaus** [comment]
But I guess the schema is generated so wonder where that generator lives?

**Comment by jstrachan** [comment]
here's the generator which runs against the OpenShift go source to generate accurate json schemas (which we use in hawtio and to generate the Java DTOs in fabric8's kubernetes-api module)
https://github.com/fabric8io/origin-schema-generator

**Comment by davsclaus** [comment]
Ah thanks, yeah maybe its for @rawlingsj to see if we can add that default value?

**Comment by jimmidyson** [comment]
Fixed by fabric8io/origin-schema-generator@7ac9aa41bdaaaccc44ace7e8055f925a9d39ef30

**Comment by jstrachan** [comment]
looks to be fixed
